ETC.tsv

```diff
@@ -2317,7 +2317,7 @@ ETC_20150317_002316	Failed the quest due to timeout! {nl} This quest can not be
 ETC_20150317_002317	Using the sentence of space
 ETC_20150317_002318	Use it in a surrounding pillar which has a similar shape.
 ETC_20150317_002319	The barrier has already been released!
-ETC_20150317_002320	Get Suteisu to worship the die Mina!
+ETC_20150317_002320	Get Suteisu to worship the Die Mina!
```

Comment:
Get Suteisu to worship the Die Mina! -> Receive status by worshipping the statue of Zemyna!